package.json

```diff
@@ -21,8 +21,6 @@
   "dependencies": {
     "@material-ui/core": "^4.11.3",
     "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.57",
-    "@material-ui/styles": "^4.11.3",
```

Comment:
This may cause issues, can you merge main into your branch before we approve this?